routes/account.rb

```diff
@@ -2,8 +2,37 @@
 
 class Clover
   hash_branch("account") do |r|
-    r.get web? do
-      r.redirect "/account/change-password"
+    r.web do
+      r.get true do
+        r.redirect "/account/multifactor-manage"
+      end
+
+      r.on "login-method" do
+        r.get true do
+          @identities = current_account.identities.to_h { [_1.provider, _1.uid] }
+
+          view "account/login_method"
+        end
+
+        r.post "disconnect" do
+          provider, uid = typecast_params.nonempty_str(["provider", "uid"])
+          identities = current_account.identities
+          unless identities.length > 1
+            # YYYY: We can allow to disconnect the last omniauth provider if the user has a password
+            # But need https://github.com/jeremyevans/rodauth/pull/461 to be merged first.
```

Comment:
It has been merged, so we can now allow disconnecting the last omniauth provider, though we'll need to update the rodauth checkout hash in the Gemfile.

Reply:
Since you've already reviewed this PR, I've created a new one for it.

https://github.com/ubicloud/ubicloud/pull/2485